grml2usb

```diff
@@ -512,28 +512,13 @@ def check_boot_flag(device):
         if part is None:
             raise HodorException("parted could not find partition")
         if part.getFlag(parted.PARTITION_BOOT):
```

Comment:
It seems like this line does not do what we expect. But it does not matter if this line is true or false. As long as the `import parted` does not throw a `ImportError` this functions returns happily.

I think we need an `else` part with:
```
            logging.debug("bootflag is NOT enabled")
            raise VerifyException(
                "Device %s does not have the bootflag set. "
                "Please enable it to be able to boot." % device
            )
```

Reply:
Thanks for the analysis and the detailed report!

Took care of all your comments in https://github.com/grml/grml2usb/pull/29/commits/705a96b84994905fbc63e3926a5af785ac0c03bc - please review once again :)